dts/arm/st/l4/stm32l4r5.dtsi

```diff
@@ -212,6 +212,19 @@
 			};
 		};
 
+		dmamux1: dmamux@40020800 {
+			compatible = "st,stm32-dmamux";
+			#dmamux-cells = <4>;
```

Comment:
where is #dmamux-cells used?  Don't see anything in the bindings.

Reply:
now changed based on the dmamux bindings from linux